whipper/common/mbngs.py

```diff
@@ -58,13 +58,13 @@ class TrackMetadata(object):
 
 class DiscMetadata(object):
     """
-    @param artist:       artist(s) name
-    @param sortName:     release artist sort name
-    @param release:      earliest release date, in YYYY-MM-DD
-    @type  release:      unicode
-    @param title:        title of the disc (with disambiguation)
-    @param releaseTitle: title of the release (without disambiguation)
-    @type  tracks:       C{list} of L{TrackMetadata}
+    :param artist:       artist(s) name
+    :param sortName:     release artist sort name
+    :param release:      earliest release date, in YYYY-MM-DD
+    :type  release:      unicode
+    :param title:        title of the disc (with disambiguation)
+    :param releaseTitle: title of the release (without disambiguation)
+    :type  tracks:       list(TrackMetadata)
```

Comment:
This seems to be on the border between this and https://github.com/whipper-team/whipper/pull/389 – I won’t object to it being here though, even if I’d like to see the reST or Sphinx documentation for the syntax. :)